src/main/java/frc/robot/subsystems/superstructure/arm/Arm.java

```diff
@@ -125,18 +128,18 @@ private boolean atPositionSetpoint() {
     }
 
     private boolean atPivotLowerLimit() {
-        return inputs.pivotLowerLimitSwitch
-                || inputs.leftPivotPositionRots <= pivotSoftLowerLimit.pivotPositionRots
+        return inputs.leftPivotPositionRots <= pivotSoftLowerLimit.pivotPositionRots
```

Comment:
You can prolly use Math.min() here for the left and right pivot rots

Reply:
Is there a reason to? Its an extra import + method call for no reason, right?